src/component/feature/FeatureToggleList/FeatureToggleListItem/FeatureToggleListItem.jsx

```diff
@@ -31,11 +32,25 @@ const FeatureToggleListItem = ({
 }) => {
     const styles = useStyles();
 
+    const { projects } = useProjects();
     const isArchive = !!revive;
 
     const { name, description, type, stale, createdAt, project, lastSeenAt } =
         feature;
 
+    const isProjectDeleted = () => {
```

Comment:
Perhaps rename this to `projectExists`? The way it reads right now is a bit confusing because you are doing a conditional in reviveFeature based on the result of calling isProjectDeleted, usually you would expect a function worded like this to return true for the condition it's checking. 

If I'm reading reviveFeature now it looks like if the proejct is deleted the function will revive the feature, which is the opposite of what is happening.